### PR TITLE
ffmpeg-tools: update ffmpegx, opus, gnutls [le9.2]

### PR DESCRIPTION
**Pull request by antonlacon**
The LE9.2 version of #3884.

### CODE REVIEW

File: packages/security/gnutls/package.mk [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 
 PKG_NAME="gnutls"
-PKG_VERSION="3.6.5"
-PKG_SHA256="073eced3acef49a3883e69ffd5f0f0b5f46e2760ad86eddc6c0866df4e7abb35"
+PKG_VERSION="3.6.10"
+PKG_SHA256="b1f3ca67673b05b746a961acf2243eaae0ffe658b6a6494265c648e7c7812293"
 PKG_LICENSE="LGPL2.1"
 PKG_SITE="https://gnutls.org"
-PKG_URL="https://www.gnupg.org/ftp/gcrypt/gnutls/v${PKG_VERSION%.*}/gnutls-$PKG_VERSION.tar.xz"
+PKG_URL="https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-$PKG_VERSION.tar.xz"
```
Comment (CvH) [inline packages/security/gnutls/package.mk]: is `v${PKG_VERSION%.*}` not working anymore ?

File: packages/addons/addon-depends/ffmpegx/package.mk [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 
 PKG_NAME="ffmpegx"
-PKG_VERSION="4.1"
-PKG_SHA256="7afb163d6974693cdad742aa1224c33683c50845c67ee5ae35506efc631ac121"
+PKG_VERSION="4.1.4"
+PKG_SHA256="f1f049a82fcfbf156564e73a3935d7e750891fab2abf302e735104fd4050a7e1"
 PKG_LICENSE="LGPLv2.1+"
 PKG_SITE="https://ffmpeg.org"
-PKG_URL="https://github.com/FFmpeg/FFmpeg/archive/n${PKG_VERSION}.tar.gz"
+PKG_URL="https://ffmpeg.org/releases/ffmpeg-$PKG_VERSION.tar.xz"
```
Comment (CvH) [inline packages/addons/addon-depends/ffmpegx/package.mk]:
```suggestion
PKG_URL="https://ffmpeg.org/releases/ffmpeg-${PKG_VERSION}.tar.xz"
```
?

File: packages/security/gnutls/package.mk (hunk 1 above)
Comment (antonlacon) [reply packages/security/gnutls/package.mk]: gnutls occasionally uses a 4th tuple (ie 3.6.7.1) which that broke on.

File: packages/addons/addon-depends/ffmpegx/package.mk (hunk 2 above)
Comment (antonlacon) [reply packages/addons/addon-depends/ffmpegx/package.mk]: Did not use the ${} because it would be the only place to do so in the file.

**Pull request merged**